tests/e2e/testfiles/storage_csm_powerscale_observability_val2.yaml

```diff
@@ -0,0 +1,488 @@
+      # enabled: Enable/Disable replication feature
+      # Allowed values:
+      #   true: enable replication feature(install dell-csi-replicator sidecar)
+      #   false: disable replication feature(do not install dell-csi-replicator sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.9.0
+      components:
+      - name: dell-csi-replicator
+        # image: Image to use for dell-csi-replicator. This shouldn't be changed
+        # Allowed values: string
+        # Default value: None
+        image: dellemc/dell-csi-replicator:v1.9.0
+        envs:
+          # replicationPrefix: prefix to prepend to storage classes parameters
+          # Allowed values: string
+          # Default value: replication.storage.dell.com
+          - name: "X_CSI_REPLICATION_PREFIX"
+            value: "replication.storage.dell.com"
+          # replicationContextPrefix: prefix to use for naming of resources created by replication feature
+          # Allowed values: string
+          # Default value: powerstore
+          - name: "X_CSI_REPLICATION_CONTEXT_PREFIX"
+            value: "powerscale"
+
+      - name: dell-replication-controller-manager
+        # image: Defines controller image. This shouldn't be changed
+        # Allowed values: string
+        image: dellemc/dell-replication-controller:v1.9.0
+        envs:
+          # TARGET_CLUSTERS_IDS: comma separated list of cluster IDs of the targets clusters. DO NOT include the source(wherever CSM Operator is deployed) cluster ID
+          # Set the value to "self" in case of stretched/single cluster configuration
+          # Allowed values: string
+          - name: "TARGET_CLUSTERS_IDS"
+            value: "self"
+          # Replication log level
+          # Allowed values: "error", "warn"/"warning", "info", "debug"
+          # Default value: "debug"
+          - name: "REPLICATION_CTRL_LOG_LEVEL"
+            value: "debug"
+
+          # replicas: Defines number of controller replicas
+          # Allowed values: int
+          # Default value: 1
+          - name: "REPLICATION_CTRL_REPLICAS"
+            value: "1"
+          # retryIntervalMin: Initial retry interval of failed reconcile request.
+          # It doubles with each failure, upto retry-interval-max
+          # Allowed values: time
+          - name: "RETRY_INTERVAL_MIN"
+            value: "1s"
+          # RETRY_INTERVAL_MAX: Maximum retry interval of failed reconcile request
+          # Allowed values: time
+          - name: "RETRY_INTERVAL_MAX"
+            value: "5m"
+
+      - name: dell-replication-controller-init
+        # image: Defines replication init container image. This shouldn't be changed
+        # Allowed values: string
+        image: dellemc/dell-replication-init:v1.0.0
+
+    # observability: allows to configure observability
+    - name: observability
+      # enabled: Enable/Disable observability
+      enabled: true
+      configVersion: v1.8.0
```

Comment:
The observability config version in file val2/CR2 is lower than that of in file val1/CR1. Should this be reversed for upgrade scenario?

Reply:
yeah i agree with santosh, Please aslo try to upgrade/downgrade to major versions and not minor versions.
Add two E2e test, one for upgrading and the other one for downgrading. You can use the same upgrade method to validate downgrade as well. Please check for downgrade E2E test added in scenarios.yaml

Reply:
> The observability config version in file val2/CR2 is lower than that of in file val1/CR1. Should this be reversed for upgrade scenario?

FIxed in New PR https://github.com/dell/csm-operator/pull/637

Reply:
> yeah i agree with santosh, Please aslo try to upgrade/downgrade to major versions and not minor versions. Add two E2e test, one for upgrading and the other one for downgrading. You can use the same upgrade method to validate downgrade as well. Please check for downgrade E2E test added in scenarios.yaml

I addressed these comments in new PR https://github.com/dell/csm-operator/pull/637.
Based feature acceptance criteria  i did  upgrade , if you feel need for downgrade please create a story